horace_core/sqw/@sqw/mask_random_pixels.m

```diff
@@ -25,7 +25,8 @@
 % Modifications: R. A. Ewings
 
 if ~has_pixels(win)
-    error('Can mask pixels only in an sqw-type object')
+    error('HORACE:sqw:invalid_argument', ...
+        'Can mask pixels only in an sqw-type object')
```

Comment:
This is kind of by definition an `sqw-type` object. This error message is now confusing. REfer to having zero pixels instead